extensions/skyportal/test_config.yaml

```diff
@@ -130,3 +130,5 @@ analysis_services:
     port: 6802
   spectral_cube_analysis_service:
     port: 7003
+
+default_groups: []
```

Comment:
I think this goes in a `user` section of the config @camilledzt (I suggest just double checking the icare default yaml or the skyportal one directly).